fineract-provider/src/main/java/org/apache/fineract/portfolio/charge/api/ChargesApiResourceSwagger.java

```diff
@@ -200,7 +200,7 @@ static final class GetChargesTemplateLoanChargeTimeTypeOptions {
             private GetChargesTemplateLoanChargeTimeTypeOptions() {}
 
             @Schema(example = "2")
-            public Long id;
```

Comment:
This looks wrong to me...It should be Long based on the EnumOptionData line:36

Reply:
Done.